pagerduty/resource_pagerduty_extension.go

```diff
@@ -44,6 +47,12 @@ func resourcePagerDutyExtension() *schema.Resource {
 				ForceNew: true,
 				Required: true,
 			},
+			"config": {
+				Type:             schema.TypeString,
+				Optional:         true,
+				ValidateFunc:     validation.ValidateJsonString,
+				DiffSuppressFunc: jsonExtensionConfigDiffSuppress,
```

Comment:
Would it be possible to use https://github.com/hashicorp/terraform/blob/master/helper/structure/suppress_json_diff.go#L9 here?

Reply:
Good catch, done.